keyboards/doio/kb16/rev2/rules.mk

```diff
@@ -5,14 +5,8 @@ SRC += ./lib/logo.c
 MCU = STM32F103
 
 # Bootloader selection
-# Cannot use `BOOTLOADER = stm32duino` due to the need to override
-# `MCU_LDSCRIPT`, therefore all parameters need to be specified here manually.
-OPT_DEFS += -DBOOTLOADER_STM32DUINO
-MCU_LDSCRIPT = STM32F103xB_stm32duino_bootloader
-BOARD = STM32_F103_STM32DUINO
-BOOTLOADER_TYPE = stm32duino
-DFU_ARGS = -d 1EAF:0003 -a 2 -R
-DFU_SUFFIX_ARGS = -v 1EAF -p 0003
+BOOTLOADER = stm32duino
```

Comment:
I guess these could be moved to info.json now.

Reply:
could do?